wix/Build.OpenJDK_generic.cmd

```diff
@@ -323,7 +324,7 @@ FOR %%A IN (%ARCH%) DO (
     )
 
     REM Add all supported languages to MSI Package attribute
-    CSCRIPT "%ProgramFiles(x86)%\Windows Kits\%WIN_SDK_MAJOR_VERSION%\bin\%WIN_SDK_FULL_VERSION%\x64\WiLangId.vbs" //Nologo ReleaseDir\!OUTPUT_BASE_FILENAME!.msi Package !LANGIDS!
+    CSCRIPT "%ProgramFiles(x86)%\Windows Kits\%WIN_SDK_MAJOR_VERSION%\bin\%WIN_SDK_FULL_VERSION%\!PLATFORM!\WiLangId.vbs" //Nologo ReleaseDir\!OUTPUT_BASE_FILENAME!.msi Package !LANGIDS!
```

Comment:
Is it still ProgramFiles(x86) on a Windoews Aarch64 machine?

Reply:
Yes! The `Windows Kits` directory is always installed to this path by default